### PR TITLE
Move site to Netlify

### PR DESCRIPTION
**Pull request by jelovirt**
## Description
Change site be generated with JavaScript during NPM install and deploy to Netlify.

## Motivation and Context
The team goal is to distribute our assets out from GitHub to remove dependency to GitHub.

Jekyll offers very little support for generating e.g. index and compound files from the source JSON files, whereas using plain JavaScript to generate the required files during NPM install is trivial. This will allow us to refactor the source files to not need to match the result files.

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# DITA Open Toolkit Plug-in Registry
+# DITA Open Toolkit Plug-in Registry [![Netlify Status](https://api.netlify.com/api/v1/badges/626192a4-0b63-4a52-9b5a-ef6b46dc6464/deploy-status)](https://app.netlify.com/sites/gracious-snyder-3a4d31/deploys)
```
Comment (infotexture) [inline ReadMe.md]: I would prefer to rename the autogenerated Netlify site to use something more indicative of its actual purpose.

Comment (jelovirt) [reply ReadMe.md]: Sure, we will be using custom plugins.dita-ot.org for this in the end.

Comment (jelovirt) [reply ReadMe.md]: Renamed to plugins-dita-ot

**Comment by jelovirt** [comment]
@infotexture Instead of using `.travis/deploy.js` for the deploy script, we could have

File: .netlify/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "registry",
+  "version": "1.0.0",
+  "description": "DITA-OT registry deployment",
+  "repository": "github:dita-ot/repository",
```
Comment (infotexture) [inline .netlify/package.json]: Looks like this should be `dita-ot/registry`

File: .travis/package.json [hunk 3]
```diff
@@ -2,11 +2,9 @@
   "name": "registry",
   "version": "1.0.0",
   "description": "DITA-OT registry validation",
+  "repository": "github:dita-ot/repository",
```
Comment (infotexture) [inline .travis/package.json]: `dita-ot/registry` ?

**Comment by jelovirt** [comment]
Fixed

**Comment by jelovirt** [comment]
AFAICT, these changes should make it possible to deploy to both GitHub Pages and Netflify, so we can have the two deployments side by side.

**Pull request merged**